packages/nx-plugin/src/executors/playwright/executor.ts

```diff
@@ -114,7 +115,8 @@ function createArgs(
  */
 function runPlaywright(args: string[], cwd: string) {
   try {
-    const cli = require.resolve('@playwright/experimental-ct-core/cli');
+    const module = require.resolve('@playwright/experimental-ct-react');
```

Comment:
`cli` module isn't part of package.json exports, so we cannot access it as before anymore